datatypes/src/operations/image/colorizer.rs

```diff
@@ -265,6 +268,7 @@ impl Colorizer {
             Self::Palette {
                 colors,
                 no_data_color,
+                default_color: _,
```

Comment:
Isn't it meaningless to have a `default_color` that isn't used?

Reply:
I think a default color makes sense, because there may be values that are neither defined in the palette nor the no data value. Thus the Platte has to be adjusted, should I do it as part of this task though?

Reply:
It isn't about having a default color, but this is ignored here and thus, it is useless. I guess you should incorporate it into the palette.

Reply:
362b10755e22f0af0999cec496d7e1bc379832a6

Reply:
I somehow can't refer to it properly, but you now output the default color rather then the no data color color when you have NaN as palette value.

Reply:
cf our discussion